components/profil/profile-header.tsx

```
@@ -0,0 +1,41 @@
+                                fill
+                                className="object-cover"
+                            />
+                        </div>
+                    ) : (
+                        <div className="w-full h-full rounded-full bg-white/10 ring-4 ring-white/50 flex items-center justify-center">
+                            <span className="text-4xl font-semibold">
+                                {session?.user?.name?.[0] || "U"}
+                            </span>
+                        </div>
+                    )}
+                    <button className="absolute bottom-0 right-0 bg-white rounded-full p-2 shadow-lg hover:bg-gray-100 transition-colors">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)